lix/packages/sdk/src/file-handlers.ts

```diff
@@ -157,98 +101,41 @@ export async function handleFileChange(args: {
 	}
 
 	await args.db.transaction().execute(async (trx) => {
-		for (const { diffs, pluginKey, pluginDiffFunction } of pluginDiffs) {
+		for (const { diffs, pluginKey } of pluginDiffs) {
 			for (const diff of diffs ?? []) {
 				// assume an insert or update operation as the default
 				// if diff.neu is not present, it's a delete operationd
 				const value = diff.neu ?? diff.old;
 
 				// TODO: save hash of changed fles in every commit to discover inconsistent commits with blob?
 
-				const previousUncomittedChange = await trx
+				// TODO make more resilient
+
+				const parentChange = await trx
```

Comment:
Plugins likely need to report the parent change. This uses ids and the date of the change. is flaky but works